src/connection_string.ts

```diff
@@ -975,6 +977,7 @@ export const OPTIONS = {
           rpOpts
         );
       }
+      throw new MongoParseError(`Unknown ReadPreference value: ${value}`);
```

Comment:
Could we add a test for this new error message if it's not covered in our spec tests already?

Reply:
Yes should be a quick unit test, good point